libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts

```diff
@@ -219,18 +219,4 @@ export class FinanceResolver {
       user,
     )
   }
-
-  @Query(() => DebtLessCertificateModel)
```

Comment:
Might want to remove the `DebtLessCertificateModel` and  `PropertyOverviewWithDetail` [here](https://github.com/island-is/island.is/pull/8651/files#diff-da7f1c60edfff63dc86178a101fda4a2879e8d08f16e593523d6f4396a8ad637L79) since they are not used in these resolvers anymore 😃